pages/latest/[content_type].tsx

```diff
@@ -56,7 +56,7 @@ export const getServerSideProps: GetServerSideProps<Props, ContextParams> = asyn
 		variables,
 	};
 
-	const { data, error, loading } = await getApolloClient( context ).query<ContentTypeByNameQuery>( queryOptions );
```

Comment:
The removal of loading can be seen in all the pages. I haven't found an alternative to this. It seems like it exists when useQuery is called, but not here? I don't quite get the logic behind that?